frontend/src/components/common/progress-details/ProgressDetails.tsx

```diff
@@ -107,15 +109,34 @@ const ProgressDetails = ({
             <Moment format="LLL" interval={0}>
               {updatedAt}
             </Moment>
+            .{' '}
+            <span onClick={handleRefreshStats} className={styles.linkLike}>
+              Refresh statistics
+            </span>
           </span>
-          <PrimaryButton onClick={handleRefreshStats}>
-            Refresh stats
-          </PrimaryButton>
         </div>
       )
     }
   }
 
+  async function exportUnsubscribers() {
+    const list = await exportEmailUnsubscribers(id)
+
+    const headers = ['Recipient', 'Reason', 'Unsubscribed At']
+    const rows = list.map((r) => [r.recipient, r.reason, r.unsubscribedAt])
+    const content = [headers, ...rows].map(toCsvRow).join('\n')
+    const exportedAt = new Date()
+
+    download(
+      new Blob([content]),
+      `${name}_unsubscribers_${exportedAt.toLocaleDateString()}_${exportedAt.toLocaleTimeString()}.csv`,
```

Comment:
maybe a bit extra, but might we want to convert the name to snake_case for consistency (think ppl won't usually name their campaigns that way)

e.g. of my file name `test unsub reason_unsubscribers_23_08_2022_19_17_55`

Reply:
also prefer date format of YYYY_MM_DD etc. but that's just me

Reply:
This is following the existing convention from our delivery report. It's been working so far (and has implied training effect on our users) so I don't think we need to stray from

Reply:
ohh ok makes sense